src/components/about.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import aboutStyles from "../styles/about.module.css";
+import Resume from "../statics/resume.json";
+
+export default function About() {
+
+  return (
+    <div className={aboutStyles.container}>
+        <div className={aboutStyles.innerContainer} id={aboutStyles.words}>
+            <h1 className={aboutStyles.font}>ABOUT</h1>
+            <h5 className={aboutStyles.font}>{Resume.basics.about}</h5>
```

Comment:
Use destructuring, ie. `const { basics } = Resume`. Try not to use a pattern like `a.b.c.d`